pysimplesoap/client.py

```diff
@@ -37,6 +37,7 @@
                      get_local_name, get_namespace_prefix, TYPE_MAP, urlsplit, \
                      iteritems
 from .wsse import UsernameToken
+from urllib3.util import make_headers
```

Comment:
The appears to force urllib3 make_headers. Is this intended to work with all transports? Is there a more generic approach to this?